hw/ip/i2c/data/i2c.hjson

```diff
@@ -506,11 +547,11 @@
       swaccess: "rw"
       hwaccess: "hro"
       fields: [
-        { bits: "15:0"
+        { bits: "12:0"
```

Comment:
With a maximum core clock of 1 GHz, all the 13-bit counters are sized to handle at least 5 us.

Reply:
Done.